webplugin/js/app/kommunicate-ui.js

```diff
@@ -440,62 +446,148 @@ setAvailabilityStatus : function (status){
+    isConversationClosed && kommunicateCommons.modifyClassList({
+        class: ["mck-box-form"]
+    }, "n-vis");
+    if (
+        isCSATenabled &&
+        isConversationClosed &&
+        !kommunicateCommons.isConversationClosedByBot() &&
+        !isCSATtriggeredByCustomer &&
+        !isConvJustResolved
+    ) {
         mckUtils.ajax({
             type: 'GET',
-            url: Kommunicate.getBaseUrl() + '/feedback' + '/'+ CURRENT_GROUP_DATA.tabId ,
+            url: Kommunicate.getBaseUrl() + '/feedback' + '/' + CURRENT_GROUP_DATA.tabId,
             global: false,
             contentType: 'application/json',
             success: function (data) {
                 var feedback = data.data
                 CURRENT_GROUP_DATA.currentGroupFeedback = feedback;
-                kommunicateCommons.modifyClassList( {class : ["mck-box-form"]}, "n-vis");
-                kommunicateCommons.modifyClassList( {class : ["mck-csat-text-1"]}, "","n-vis");
-                kommunicateCommons.modifyClassList( {id : ["mck-sidebox-ft"]}, "mck-closed-conv-banner");
-                kommunicateCommons.modifyClassList( {id : ["csat-1","csat-2","csat-3"]}, "n-vis");
+                kommunicateCommons.modifyClassList({
+                    class: ["mck-box-form"]
+                }, "n-vis");
+                kommunicateCommons.modifyClassList({
+                    class: ["mck-csat-text-1"]
+                }, "", "n-vis");
+                kommunicateCommons.modifyClassList({
+                    id: ["mck-sidebox-ft"]
+                }, "mck-closed-conv-banner");
+                kommunicateCommons.modifyClassList({
+                    id: ["csat-1", "csat-2", "csat-3"]
+                }, "n-vis");
+                kommunicateCommons.modifyClassList({
+                    id: ["km-widget-options"]
+                }, "n-vis");
                 /*
                 csat-1 : csat rating first screen where you can rate via emoticons.
                 csat-2 : csat rating second screen where you can add comments.
                 csat-3 : csat result screen where you show overall feedback.
                 */
                 if (feedback && feedback.rating) {
-                    if(feedback.comments.length > 0){ // if comments are there in feedback 
-                        kommunicateCommons.modifyClassList( {id : ["csat-3","mck-rated"]}, "", "n-vis");
-                        document.getElementById('csat-3').innerHTML = '\"' + feedback.comments[0] + '\"';
-                    } else { // only rating via emoticons
-                        kommunicateCommons.modifyClassList( {id : ["csat-2","mck-rated"]}, "", "n-vis");
-                    }
-                    document.getElementById('mck-rating-container').innerHTML = kommunicateCommons.getRatingSmilies(feedback.rating);
+                    // if(feedback.comments.length > 0){ // if comments are there in feedback 
```

Comment:
is this logic moved somewhere else?

Reply:
This portion of code is to be ignored so made the suitable changes